src/main/java/teammates/ui/pagedata/PageData.java

```diff
@@ -858,4 +862,97 @@ public void setStatusMessagesToUser(List<StatusMessage> statusMessagesToUser) {
         return statusMessagesToUser;
     }
 
+    public FeedbackResponseCommentRow buildFeedbackResponseCommentAddForm(FeedbackQuestionAttributes question,
+            String responseId, Map<FeedbackParticipantType, Boolean> responseVisibilityMap,
+            String giverName, String recipientName, boolean isInstructor, double timeZone) {
+        FeedbackResponseCommentAttributes frca =
+                new FeedbackResponseCommentAttributes(question.courseId, question.feedbackSessionName,
+                                                      question.getFeedbackQuestionId(), responseId);
```

Comment:
More custom indentation issues. I'll avoid commenting any further on this for this review.